app/src/main/java/com/example/tudee/presentation/components/OnBoardingPage.kt

```diff
@@ -28,109 +26,108 @@ import com.example.tudee.designsystem.theme.TudeeTheme
 import com.example.tudee.presentation.components.buttons.FabButton
 import com.example.tudee.presentation.screen.onboarding.OnBoardingPageUiModel
 
+
 @Composable
 fun OnBoardingPage(
-    modifier: Modifier = Modifier,
     onClick: () -> Unit = {},
     orientation: Boolean = true,
     onBoardingPageUiModel: OnBoardingPageUiModel,
 ) {
     if (orientation) {
         Column(
-            modifier = modifier.padding(horizontal = 16.dp),
-            verticalArrangement = Arrangement.spacedBy(16.dp),
+            modifier = Modifier
+                .padding(horizontal = 16.dp),
+            verticalArrangement = Arrangement.Center,
             horizontalAlignment = Alignment.CenterHorizontally
         ) {
-            OnBoardingPageContent(onBoardingPageUiModel, onClick)
+            OnBoardingPageContent(Modifier, onBoardingPageUiModel, onClick)
```

Comment:
What's the point of passing the default `Modifier` to a private composable with an already default parameter with this value?